test/Chainweb/Test/Pact/RemotePactTest.hs

```diff
@@ -175,8 +177,9 @@ spvRequests nio = testCaseSteps "spv client tests"$ \step -> do
       let pm = CM.PublicMeta (CM.ChainId "0") "sender00" 100000 0.01 100000 0
 
       tid <- mkChainId version (1::Int)
-      cmd <- liftIO $ mkExec txcode (txdata tid) pm ks (Just "0")
-      return $ SubmitBatch (pure cmd)
+      cmd1 <- liftIO $ mkExec txcode (txdata tid) pm ks (Just "0")
+      cmd2 <- liftIO $ mkExec txcode (txdata tid) pm ks (Just "1")
```

Comment:
N.B. putting multiple txs in the block is important for failing the test

Reply:
Thanks